rmw_fastrtps_cpp/CMakeLists.txt

```diff
@@ -53,7 +54,7 @@ ament_target_dependencies(rmw_fastrtps_cpp
 
 configure_rmw_library(rmw_fastrtps_cpp)
 
-ament_export_libraries(rmw_fastrtps_cpp fastcdr fastrtps)
+ament_export_libraries(rmw_fastrtps_cpp fastcdr fastrtps ${FastRTPS_LIBRARIES})
```

Comment:
This line still seems to export the libraries twice. I don't understand why that should be the case.